libgnucash/app-utils/options.scm

```diff
@@ -18,6 +18,12 @@
 ;; Boston, MA  02110-1301,  USA       gnu@gnu.org
 (use-modules (ice-9 regex))
 
+(define (rpterror-earlier type newoption fallback)
+  (let ((msg (format #f "This report was saved using a later version of GnuCash. \
+One of the newer ~a options '~a' is not available, fallback to the option \
+'~a'." type newoption fallback)))
+    (gnc:gui-warn msg)))
```

Comment:
This is a message that will be shown to the end user so it should be translatable...